### DISCUSSION TITLE
Debug Symbols seem not being stripped from xcframework

### DISCUSSION THREAD

**Discussion by jesus-mg-ios** [airbnb/lottie-ios#2443] [General]
## Which Version of Lottie are you using?

Lottie 4.4.3

## Expected Behavior 
Debug Symbols stripped from framework and DSYMs delivered in tags. 

## Actual Behavior
Debugs Symbols are in the framework.
`Use strip -rSTx <binary>` after generating DSYMS -> `dsymutil <binary> -o <binary>.dSYM` to decrease framework size.

**Comment by calda** [comment]
Consumers asked us to include the dSYMs in the XCFramework so you could symbolicate crashes: https://github.com/airbnb/lottie-ios/discussions/2282

**Reply by jesus-mg-ios** [reply]
I think that if they are present outside, (uploaded in tag section) shouldn't be a problem for other consumers.
But if you want you can do both:

**Two Scenarios**: dSYMs Inside vs. Outside the Binary
dSYMs Inside the Binary

**Embedded dSYMs**: Sometimes, debug symbols are included directly within the binary. This makes the binary larger but simplifies the symbolication process because the debug information is always present with the binary.
* **Pros**: Easier to manage for internal testing and development, as the binary carries its own debug information.
* **Cons**: Increased binary size, which might not be ideal for production releases.
dSYMs Outside the Binary

**Separate dSYMs**: Typically, in production, dSYMs are generated separately from the binary and need to be managed independently.
Symbolication Process: For services like Crashlytics, you must upload the dSYM files. When a crash occurs, the service uses these uploaded dSYMs to symbolicate the crash reports.
* **Pros**: Keeps the binary size smaller and more efficient for distribution. Enhanced security as debug information is not shipped with the binary.
* **Cons**: Requires careful management of dSYM files. You need to ensure that the correct dSYM file is available for each build/version of your app.
